biz.aQute.bndlib/src/aQute/bnd/osgi/repository/BridgeRepository.java

```diff
@@ -318,6 +354,12 @@ public static void addInformationCapability(ResourceBuilder rb, String name, Str
 	public static void addInformationCapability(ResourceBuilder rb, String name, Version version, String from,
 		String error) {
 
+		addInformationCapability(rb, name, version, from, error, null, null, null, null, null);
+	}
+
+	public static void addInformationCapability(ResourceBuilder rb, String name, Version version, String from,
+		String error, String groupId, String artifactId, String mavenVersion, String extension, String classifier) {
```

Comment:
Instead of adding all these parameters for maven info, consider a map and passing `archive.attributes()`

https://github.com/bndtools/bnd/blob/7a5b449d1469cedefc10ecd8c7ec6292e91e6def/biz.aQute.repository/src/aQute/maven/api/Archive.java#L246

Then you can just add the map to the cap attributes.

Reply:
same question as above.